### PR TITLE
Enabled on_element_click for Graph mark

### LINKED ISSUE #500: "on_element_click()" does not seem to work for Graph mark
When constructing a Graph mark, it seems that I can set a callback function using, for example, `graph.on_background_click(print_event)` but setting a callback using `graph.on_click()` or `graph.on_element_click()` does not seem to do anything.

Really loving the Graph mark! Thank you for all your great work!!

### PR DESCRIPTION
**Pull request by ChakriCherukuri**
fixes #500

### CODE REVIEW

**Comment by ChakriCherukuri** [comment]
@dustinmichels please review

File: examples/Marks/Graph.ipynb [hunk 1]
```diff
@@ -197,56 +244,108 @@
...
+   "metadata": {
+    "collapsed": true,
+    "deletable": true,
+    "editable": true
+   },
    "outputs": [],
    "source": [
-    "# click on any node to select it (selected attribute gets populated with the selected node)\n",
-    "graph.selected"
+    "import bqplot.pyplot as plt\n",
```
Comment (ssunkara1) [inline examples/Marks/Graph.ipynb]: I think we can move this to the cell right after the tooltip. I think that feels better

**Comment by ssunkara1** [comment]
Thanks @ChakriCherukuri Apart from the small notebook comment, it looks good.

**Comment by ChakriCherukuri** [comment]
@ssunkara1 made changes per your recommendation. Please review

**Pull request merged**